library/Fission/User/Password.hs

```diff
@@ -4,7 +4,9 @@ import           Fission.Prelude
 import           Fission.Random              as Random
 import qualified Fission.User.Password.Types as User
 
+{-| Generate a password for a `User`.
+-}
```

Comment:
Not a fan of `-- |`?

Reply:
Doesn't bother me. I started writing a doc test for this and ended up removing it, so that's why it looks like this.